tensorflow/core/kernels/slice_op_cpu_impl_6.cc

```diff
@@ -1,18 +0,0 @@
-/* Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
I see, but I delete the template parameter "NDIM", how could I bring these files back? There is similar implementation in transpose_op https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/transpose_functor_cpu.cc#L55